src/ucs/sys/topo/base/topo.c

```diff
@@ -790,3 +790,65 @@ const char *ucs_topo_resolve_sysfs_path(const char *dev_path, char *path_buffer)
               sysfs_path);
     return sysfs_path;
 }
+
+static const char *
+ucs_topo_iface_get_lower_dev_path(const struct dirent *entry,
```

Comment:
no need for this function, just add the code in `uct_topo_iface_parse_virtual_dev`